src/components/ValueTypeDropDown.vue

```diff
@@ -26,12 +18,19 @@
 import Vue from 'vue';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 
+import { Dropdown } from '@wmde/wikit-vue-components';
+
 export default Vue.extend( {
 	name: 'ValueTypeDropDown',
 	data() {
 		return {
-			selected: PropertyValueRelation.Matching,
-			optionItems: PropertyValueRelation,
+			selectedItem: {
+				label: PropertyValueRelation.Matching,
+				description: '',
+			},
+			optionItems: Object.values( PropertyValueRelation ).map( ( value: PropertyValueRelation ) => {
```

Comment:
Those option items should have translations. They could have the form of
```
{ 
  value: PropertyValueRelation.Matching,
  label: i18n('query-builder-property-lookup-' + PropertyValueRelation.Matching ),
}
```
 or something like that

Reply:
I noticed the problem way before the upgrade, I feel this is out of scope of this PR, we should fix it soon but separately

Reply:
I think that is fair. Could you create a ticket in the current iteration board for this while I look at the other issues?